src/tests/JIT/Performance/JittedMethodsCountingTest/JittedMethodsCountingTest.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="JittedMethodsCountingTest.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="HelloWorld/HelloWorld.csproj">
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
```

Comment:
This clause is what I believe should tell the merged wrapper generator that we should skip this reference as it's not supposed to be a direct project component.

Reply:
You might also need to change the `OutputItemType` to `Content` to get it to not warn.

Reply:
Hmm, is that somehow related to the next line or something completely different?

Reply:
Sorry, didn't see that under the comment. You're right, nevermind.